docs/profiles.md

```diff
@@ -33,15 +45,75 @@ All other calculations stem from that, including the returned timings in driving
 
 Using the power of the scripting language you wouldn't typically see something as simple as a `result.forward_speed = 20` line within the way_function. Instead a way_function will examine the tagging (e.g. `way:get_value_by_key("highway")` and many others), process this information in various ways, calling other local functions, referencing the global variables and look-up hashes, before arriving at the result.
 
-## Guidance
+The following attributes can be set on the result in way_function:
+
+Attribute                               | Type     | Notes
+----------------------------------------|----------|--------------------------------------------------------------------------
+forward_rate                            | Float    | Routing weight on this way per metre
```

Comment:
The unit for `rate` is actually weirder. Speed is `meter per seconds`, this is a fake speed quantity so the unit is `meter per {weight}`.